packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard/utils.lua

```diff
@@ -153,4 +153,8 @@ function utils.append_network(dev)
     end
 end
 
+function utils.filter_alredy_scanned(hosts, results)
+    return ft.filter(function(host) return results[host.host] == nil end, hosts)
+end
+
 return utils
```

Comment:
Still I see this missing new line at EOF

Reply:
ready, Im thinking on adapting this script  https://gist.github.com/johnjohndoe/4024222 :)